src/HaroohiePals.Gui/ImGuiGameWindowSettings.cs

```diff
@@ -1,12 +1,11 @@
 ﻿#nullable enable
 
-using HaroohiePals.Gui.View.Menu;
 using OpenTK.Mathematics;
 using System.Collections.Generic;
 
 namespace HaroohiePals.Gui;
 
-public record struct ImGuiGameWindowSettings(string Title, Vector2i Size, float UiScale, IReadOnlyCollection<ImGuiIcon> AppIcons)
+public sealed record ImGuiGameWindowSettings(string Title, Vector2i Size, float UiScale, IReadOnlyCollection<ImGuiIconGlyph> IconGlyphs, IReadOnlyCollection<ImGuiFont> Fonts, ImGuiFont? IconFont = null)
```

Comment:
```suggestion
public sealed record ImGuiGameWindowSettings(
    string Title,
    Vector2i Size,
    float UiScale,
    IReadOnlyCollection<ImGuiIconGlyph> IconGlyphs,
    IReadOnlyCollection<ImGuiFont> Fonts,
    ImGuiFont? IconFont = null)
```
This line is way too long. Put each argument on a separate line.